libsql-server/src/http/user/extract.rs

```diff
@@ -15,6 +16,8 @@ impl FromRequestParts<AppState> for RequestContext {
         parts: &mut axum::http::request::Parts,
         state: &AppState,
     ) -> std::result::Result<Self, Self::Rejection> {
+
+        // start todo this block is same as the one in mod.rs
```

Comment:
is this some todo format Ive never seen before? Does this work with your editor? seems interesting!

Reply:
no just my way of tagging possible duplication, removed it